keps/sig-storage/1488-csi-migration-gce-pd/README.md

```diff
@@ -40,6 +45,38 @@ used as as described in its parent KEP. For all other contents, please refer to
 
 Please refer to the [CSI Migration Production Readiness Review Questionnaire](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration#production-readiness-review-questionnaire).
 
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+No additional tests are needed, rather the issue is orchestrating CSI driver
+deployment for prow jobs. This has been complicated by the cloud provider
+extraction work, which no longer permits cloud provider specific orchestration
+in the k/k repository. This means that it is not possible to run any test for
+gce-pd in k/k. All such tests have been moved to the [cloud-provider-gcp repo and
+test infrastructure](https://github.com/kubernetes/cloud-provider-gcp).
+
+##### Unit tests
+
+N/A
```

Comment:
Are there no unit tests?

Reply:
Like this: https://github.com/kubernetes/csi-translation-lib/blob/master/plugins/gce_pd_test.go

Reply:
Ah, thanks, I didn't know about those. Added.